Assets/MRTK/Core/Definitions/MixedRealityToolkitConfigurationProfile.cs

```diff
@@ -40,6 +40,19 @@ public ExperienceScale TargetExperienceScale
             set { targetExperienceScale = value; }
         }
 
+        [SerializeField]
+        [Tooltip("The height above the floor for the Mixed Reality Experience. 1 unit = 1 meter")]
+        private float floorHeight = 1.0f;
```

Comment:
I think it should be kept to 1.0 since there's no instance where a value of 0 results in desirable behavior with the services that use the floorHeight variable to my knowledge.

Reply:
Hmm I guess I have two concerns:
1. Will a none zero value break/change the behavior of an existing app when the dev upgrade to a new version of MRTK containing this change?
2. Is this setting only affecting VR apps but not AR ones? If so how? If not then we might want to state that in the tooltip/doc comments.

Also I just noticed that in the inspector the floor height field is between the target scale dropdown and the text block that elaborates on the chosen target scale. Maybe we want to move the floor height field a little lower so that the text block is right below the dropdown.

Reply:
As far as Unit tests/functionality goes, I don't think there should be a breaking change to existing apps, though you do make a good point in that there might be value in differentiating between AR/VR experiences as well. 

Given the rediscovery of this issue, I'm going to take another look at this with additional context: https://github.com/microsoft/MixedRealityToolkit-Unity/issues/7913